data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/ExportGDBFieldsToFolder.py

```diff
@@ -53,8 +53,20 @@ def exportFields():
 	# Set the workspace for ListFeatureClasses
 	arcpy.env.workspace = gdb
 
+	# List feature classes from this dataset name
 	featureClasses = arcpy.ListFeatureClasses(feature_dataset='MilitaryOverlay')
 
+	# Check for alternate dataset names
+	if len(featureClasses) == 0:
+		featureClasses = arcpy.ListFeatureClasses(feature_dataset='militaryoverlay2525d')
```

Comment:
@Dbarnes1 - here was the change needed to list the new feature dataset names "militaryoverlay2525d" "militaryoverlay2525b2" instead of the old one "MilitaryOverlay" and add an error to the output if that fails to return anything (nothing was happening when I tried this before this PR so had to add this bit) - HIH